tests/queries/0_stateless/02766_prql.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
Looks like this test can be converted to `.sql`, I don't see any reasons for `.sh`

Reply:
The reason is the `.sql` tests are picked up by the ASTFuzzer, but it is not prepared to handle dialects. I was advised to convert the test to a `.sh` test to avoid messing up the ASTFuzzer.

Reply:
Good point, thanks